components/navbar.tsx

```diff
@@ -14,9 +16,14 @@ function NavBar() {
         SalesWell
       </Typography>
       { session ? 
-        <Button color="inherit" id = "logout-button" onClick={() => signOut()}>
-          Logout
-        </Button>
+        <>
+          <Button color="inherit" onClick={() => router.push('/booking/list')}>
```

Comment:
we should use Link here: https://stackoverflow.com/questions/65086108/next-js-link-vs-router-push-vs-a-tag